bin/dbunloader.py

```diff
@@ -158,7 +159,7 @@ def _bounded_records_per_message(config_object, logger):
 
     interval = cp.get('unloader', 'interval')
 
-    unloader = DbUnloader(db, unload_dir, include_vos, exclude_vos, local_jobs, withhold_dns)
+    unloader = DbUnloader(db, unload_dir, include_vos, exclude_vos, local_jobs, withhold_dns, dict_records)
```

Comment:
I guess you might be having the changes ready already. It's about replicating the same in other places as well ? like client.py

Reply:
You mean in other places that need to unload?

Reply:
like in client.py, When client has unloader enabled case?